### PR TITLE
ENG-197: [PXB2.4] Introduce linking script

### PR DESCRIPTION
**Pull request by Sudokamikaze**

### CODE REVIEW

**Review by EvgeniyPatlan — COMMENTED** [review]
looks ok

**Comment by Sudokamikaze** [comment]
Changed: Reverted innodb commit, added `epel-release` to test suite

File: storage/innobase/xtrabackup/test/bootstrap.sh [hunk 1]
```diff
@@ -26,7 +26,7 @@ then
     usage
 fi
 
-arch="`uname -m`"
+arch="$(uname -m)"
```
Comment (EvgeniyPatlan) [inline storage/innobase/xtrabackup/test/bootstrap.sh]:
```suggestion
arch=$(uname -m)
```

File: storage/innobase/xtrabackup/utils/build-binary.sh [hunk 2]
```diff
@@ -91,7 +91,7 @@ export MAKE_JFLAG=-j4
 
 # Create a temporary working directory
 BASEINSTALLDIR="$(cd "$WORKDIR" && TMPDIR="$WORKDIR_ABS" mktemp -d xtrabackup-build.XXXXXX)"
-INSTALLDIR="$WORKDIR_ABS/$BASEINSTALLDIR/percona-xtrabackup-$XTRABACKUP_VERSION-$(uname -s)-$(uname -m)"   # Make it absolute
+INSTALLDIR="$WORKDIR_ABS/$BASEINSTALLDIR/percona-xtrabackup-$XTRABACKUP_VERSION-$(uname -s)-$(uname -m)$GLIBC_VER"   # Make it absolute
```
Comment (EvgeniyPatlan) [inline storage/innobase/xtrabackup/utils/build-binary.sh:94]: where is GLIBC_VER defined?

**Review by EvgeniyPatlan — CHANGES_REQUESTED** [review]

**Review by EvgeniyPatlan — APPROVED** [review]

**Review by vorsel — APPROVED** [review]

**Pull request merged**